harness/determined/cli/workspace.py

```diff
@@ -23,9 +23,12 @@
 ]
 
 
-def render_workspaces(workspaces: Sequence[bindings.v1Workspace]) -> None:
-    values = [
-        [
+def render_workspaces(
+    workspaces: Sequence[bindings.v1Workspace], from_list_api: bool = False
```

Comment:
is the purpose of `from_list_api` here to make the `det workspace list` table more concise by skipping checkpoint storage which is a large json blob?

Reply:
Yea I didn't want to print the ```checkpoint_storage_header``` in ```det workspace ls``` when all of them will be null since the API won't return them

Then I didn't want to have the ```GetWorkspaces``` route return them since it would also need to marshal all the listed checkpoint config to experiment config in order to mask any potential secrets in the config then back which could add potential noticeable impact to page loads.